girder/models/model_base.py

```diff
@@ -892,7 +1029,8 @@ def getFullAccessList(self, doc):
 
         return acList
 
-    def setUserAccess(self, doc, user, level, save=False):
+    def setUserAccess(self, doc, user, level, save=False, flags=None, currentUser=None,
```

Comment:
Since there is already a `user` parameter to `setUserAccess`, if we wanted consistency we'd have to change the one in `setGroupAccess` to be called `currentUser`. Is that OK?